### ISSUE TITLE
Can not reproduce the segmentation mIoU on ADE20k

### ISSUE THREAD

**Issue by shiyutang** [fudan-zvg/SeaFormer#3]
I try to reproduce the base model, but I only get mIoU=40.0 rather than 41.2 as reported in the codebase.
![image](https://user-images.githubusercontent.com/34859558/217713549-a10bef42-3cd0-4f18-9cb3-05223948c2d3.png)

Here are some change I do to the original code.
1. I train the model on 4 cards with 4 images for each card.
![image](https://user-images.githubusercontent.com/34859558/217694670-6f984f9d-7d9d-44c3-8251-a160a22ce602.png)

2. I load the pre-trained classification model through the config:  
![image](https://user-images.githubusercontent.com/34859558/217695071-08068070-63ee-4047-a34b-dc3a13e69661.png)

And the script I used to train is: export CUDA_VISIBLE_DEVICES=3,5,6,7;sh tools/dist_train.sh  local_configs/seaformer/seaformer_base_512x512_160k_2x8_ade20k.py  4 --work-dir output 

Could you guys point out where could be wrong? Thank you very much :)

**Comment by wwqq** [comment]
We reported mIoU=40.2 in the paper. We noticed that you have already reproduced this result. The model released in this repo is a better one after multiple training.

**Comment by shiyutang** [comment]
Thank you very much. And I wonder, did you tune anything during multiple training? or did you achieve a 1% raise in mIoU out of training fluctuation?

**Comment by wwqq** [comment]
We didn't tune anything. We used the same settings as in this repo.

**Comment by shiyutang** [comment]
Thanks a lot.

**Comment by nizhenliang** [comment]
Hello, I meet the same problem. I reproduce seaformer-base on ADR20K. But only get 39.96 mIoU. I find the lr in paper is 0.0005, but in this repo lr is 0.00025. I want to know whcih one is your config?

**Comment by wwqq** [comment]
@nizhenliang Hello, lr is 0.0005 for batch_size 32 and 0.00025 for 16

**Comment by 1787648106** [comment]
> We didn't tune anything. We used the same settings as in this repo.

hello，I wonder the value of the seed in this experiment，thanks.
